CODE_OF_CONDUCT.md

```diff
@@ -257,14 +262,13 @@ Potential consequences (for reportees) of Quansight Code of Conduct incidents in
 * A permanent or temporary ban from some or all Quansight spaces (mailing
   lists, GitHub repos, in-person events, etc.)
 * Assistance to the complainant with a report to other bodies (e.g., institutional offices or appropriate law enforcement agencies)
-* An imposed vacation or removal from Quansight employment/affiliation (for
-  Quansight personnel/affiliates)
 * Publishing an account of the incident and calling for the resignation of the
   reportee(s) from their responsibilities (usually pursued when Quansight lacks
   formal authority: this may be appropriate if the person is an external event
   leader, if they refuse to recuse themselves from a conflict of interest, or
   similar)
 * Any other response that Quansight deems necessary and appropriate to the situation
+* Referral to the appropriate HR staff for Quansight employees.
```

Comment:
moved this from lines 260-261 and rephrased it, but this may be redundant since it follows "Any other response".

Reply:
This works.